src/Symfony/Installer/SelfUpdateCommand.php

```diff
@@ -15,11 +15,12 @@
 use Symfony\Component\Console\Input\InputInterface;
 use Symfony\Component\Console\Output\OutputInterface;
 use Symfony\Component\Filesystem\Filesystem;
+use Symfony\Component\Filesystem\Exception\IOException;
 
 /**
  * This command is inspired by the self-update command included
  * in the PHP-CS-Fixer library. See:
- * https://github.com/fabpot/PHP-CS-Fixer/blob/master/Symfony/CS/Console/Command/SelfUpdateCommand.php
+ * https://github.com/fabpot/PHP-CS-Fixer/blob/master/Symfony/CS/Console/Command/SelfUpdateCommand.php.
```

Comment:
the proper fix would be to move this to the long description of the class instead